### PR TITLE
feat: Added Responsiveness to the project

### PR DESCRIPTION
**Pull request by Amrhub**
- Adjusted navbar in HTML to match template design 
- Added responsiveness in the CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -195,15 +221,15 @@ <h1 class="Section-title">
...
+                            <input type="text" name="name" id="full-name" placeholder="First and Last" required="" maxlength="30">
+                            <input type="email" name="_replyto" id="email-address" placeholder="email@domain.tld" required="">
+                            <textarea rows="5" name="message" id="message" placeholder="Enter your message..." required="" maxlength="500"></textarea>
+                            <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
                         </fieldset>
                         <div class="submit-container">
                             <input class="btn-primary" type="submit" value="Get in touch">
                         </div>
-                      </form>
+                    </form>
```
Comment (TedLivist) [inline index.html:232]: - [ ] Please improve the look of your contact form by adding the appropriate margins
- [ ] Also add the missing social media links at the bottom of the page

**Currently, you have this:**

amr-form-footer

**The Figma template**

amr-form-footer-orig

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Portfolio: mobile version - Contact form
+# Portfolio: Desktop version
 
-> Portfolio: mobile version - Contact form
+> Portfolio: Desktop version
```
Comment (TedLivist) [inline README.md:6]: - [ ] There appears to be a problem with your images. They are not broken. Please them

amr-imgs

- [ ] Your page also appears to be missing the background effects contained in the templates

amr-bckg

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Amrhub,

Good job so far! Your UI design is top-notch ✔️ 
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Amrhub) [reply README.md:6]: Not sure, I totally understood what you meant here

Comment (TedLivist) [reply README.md:6]: @Amrhub I meant that your project does not have the background images present in the Figma template.

**Review by TedLivist — APPROVED** [review]

**Review by TedLivist — APPROVED** [review]
Hi @Amrhub,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**